draft-sharabayko-srt.md

```diff
@@ -1531,113 +1544,123 @@ The INDUCTION phase serves only to set a cookie on the Listener so that it
-- Extension Field: SRT magic code 0x4A17
+- HS Version: 5.
+- Encryption Field: Advertised cipher family and block size.
+- Extension Field: SRT magic code 0x4A17.
 - Handshake Type: INDUCTION
 - SRT Socket ID: Socket ID of the Listener
 - SYN Cookie: a cookie that is crafted based on host, port and current time
   with 1 minute accuracy to avoid SYN flooding attack {{RFC4987}}.
+- There MUST be no HS extensions.
 
 At this point the Listener still does not know if the Caller is SRT or UDT,
 and it responds with the same set of values regardless of whether the Caller is
 SRT or UDT.
 
-If the party is SRT, it does interpret the values in Version and Extension Field.
-If it receives the value 5 in Version, it understands that it comes from an SRT
-party, so it knows that it should prepare the proper handshake messages
-phase. It also checks the following:
+A legacy UDT party completely ignores the values reported in the HS Version and the Handshake Type field.  
+It is, however, interested in the SYN Cookie value, as this must be passed to the next 
+phase. It does interpret these fields, but only in the "conclusion" message.
 
-- whether the Extension Flags contains the magic value 0x4A17; otherwise the
-  connection is rejected. This is a contingency for the case where someone who,
-  in an attempt to extend UDT independently, increases the Version value to 5
-  and tries to test it against SRT;
+#### The Conclusion Phase
 
-- whether the Encryption Flags contain a non-zero
-  value, which is interpreted as an advertised cipher family and block size.
+##### The Conclusion Request
 
-A legacy UDT party completely ignores the values reported in Version and Handshake Type.
-It is, however, interested in the SYN Cookie value, as this must be passed to the next
-phase. It does interpret these fields, but only in the "conclusion" message.
+The SRT caller receives the Induction Response from the SRT listener. The SRT caller MUST check the Induction response from the SRT listener.
 
-#### The Conclusion Phase
+If the HS Version value is 5, the response came from SRT, and the handshake version 5 procedure is performed
+as covered further. 
+If the HS Version value is 4, the legacy handshake procedure can be applied if supported. The procedure is deprecated and is not covered here.
+The caller MAY reject the connection with the `SRT_REJ_VERSION` reason. In this case there is nothing to send to the SRT listener, as there is no
+connection established at this point.
+
+The Extension Flags field MUST contain the magic value 0x4A17. If it does not, the
+connection MUST be rejected with rejection reason `SRT_REJ_ROGUE`. This is, among other things,
+a contingency for the case when someone, in an attempt to extend UDT independently, increases the HS Version value to 5
+and tries to test it against SRT. In this case there is nothing to send to the SRT listener, as there is no
+connection established at this point.
```

Comment:
`In this case there is nothing to send to the SRT listener, as there is no
connection established at this point.`

Duplicate of text in lines 1598-1599

Reply:
- [ ] To be removed.